site/src/pages/WorkspacePage/WorkspaceReadyPage.tsx

```diff
@@ -208,7 +208,7 @@ export const WorkspaceReadyPage: FC<WorkspaceReadyPageProps> = ({
 				hideSSHButton={featureVisibility.browser_only}
 				hideVSCodeDesktopButton={featureVisibility.browser_only}
 				buildInfo={buildInfoQuery.data}
-				sshPrefix={sshPrefixQuery.data?.hostname_prefix}
+				sshSuffix={sshConfigQuery.data?.hostname_suffix}
```

Comment:
One consequence of this change is that if operators have the prefix set to something other than `coder`, but haven't set the suffix, then their custom name won't be used in the call to action. It won't prevent the prefix from still working, but might be surprising to see `.coder` in the UI.

It can be their encouragement to set the new config option, but could generate some support tickets.

Reply:
I think we need to force-push here to promote the use of the suffix.

Reply:
I agree that we should mark CODER_SSH_HOSTNAME_PREFIX as deprecated, but that can be a separate PR.

Reply:
Filed https://github.com/coder/coder/issues/18156